src/main/java/domain/ladder/LadderGenerator.java

```diff
@@ -0,0 +1,18 @@
+package domain.ladder;
+
+import domain.user.Users;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class LadderGenerator {
+    public static Ladder generate(final Users users, int height) {
+        List<LadderLine> ladderLines = new ArrayList<>();
+
+        IntStream.range(0, height)
+                .forEach((index) -> ladderLines.add(LadderLineGenerator.generate(users)));
```

Comment:
```suggestion
        IntStream.range(0, height)
                .mapToObj(index -> LadderLineGenerator.generate(users))
                .collect(Collectiors.toList());
```
위와 같이 foreach로 하나씩 담기보다 바로 생성해서 할당하는건 어때요?

Reply:
어떻게 할까 생각하다가 잠들었는데 좋은 정보 감사합니다 ㅎㅎ